.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Sample Development Environment for CCF",
   "context": "..",
-  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.0",
+  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.1-0",
```

Comment:
Just verifying -  is this the correct version format here?

Reply:
```suggestion
  "image": "ccfmsrc.azurecr.io/ccf/ci/sgx:oe-0.18.2-0",
```

Reply:
Nope, this was wrong - good spot! Fixed now.